ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -302,6 +314,15 @@ def pytest_addoption(parser):
         loaded when run-ci starts
         """,
     )
+    parser.addoption(
+        "--install-lvmo",
+        dest="install_lvmo",
+        action="store_true",
+        default=False,
+        help="""
+            set for installing lvmo operator and lvmo cluster
+            """,
+    )
```

Comment:
please add documentation in docs/usage.md

Reply:
Documented.